lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -63,9 +64,9 @@ class PostgreSqlSchemaManager extends AbstractSchemaManager
      */
     public function getSchemaNames()
     {
-        $rows = $this->_conn->fetchAll("SELECT nspname as schema_name FROM pg_namespace WHERE nspname !~ '^pg_.*' and nspname != 'information_schema'");
+        $statement = $this->_conn->executeQuery("SELECT nspname FROM pg_namespace WHERE nspname !~ '^pg_.*' AND nspname != 'information_schema'");
 
-        return array_map(function ($v) { return $v['schema_name']; }, $rows);
+        return $statement->fetchAll(FetchMode::COLUMN);
```

Comment:
Is this something the inspections found? If so, I'm incredibly impressed

Reply:
No, it was suggested in discussion above. :)
https://github.com/doctrine/dbal/pull/3200#discussion_r199330990

Reply:
Ah thanks!